.github/workflows/CI.yml

```diff
@@ -62,7 +62,7 @@ jobs:
         logstash: [ "7.16.3" ]
         opendistro: [ "1.13.3" ]
         integration: [ true ]
-        secure: [ false ]
+        secure: [ true, false ]
```

Comment:
what does this line do - does it have the acceptable values for secure?

Reply:
it is a matrix where it creates multiple jobs with variable substitution.

Here, we are having true and false for secure matrix, to run both secured and unsecured integration jobs